web/src/modules/common/components/CleanAnchor.js

```diff
@@ -9,22 +9,17 @@ const StyledAnchor = styled.a`
 `
 
 type PropsType = {|
-  newTab?: boolean,
   href: string,
   children: React.Node,
   ariaLabel?: string,
   className?: string
 |}
 
 class CleanAnchor extends React.PureComponent<PropsType> {
-  static defaultProps = {
-    newTab: true
```

Comment:
For other reviewers: This is the line which is responsible for opening the link in the same tab